Source/Core/DolphinQt/Settings/WiiPane.cpp

```diff
@@ -120,14 +135,21 @@ void WiiPane::CreateMisc()
   m_system_language_choice->addItem(tr("Traditional Chinese"));
   m_system_language_choice->addItem(tr("Korean"));
 
-  m_pal60_mode_checkbox->setToolTip(tr("Sets the Wii display mode to 60Hz (480i) instead of 50Hz "
-                                       "(576i) for PAL games.\nMay not work for all games."));
+  m_refresh_rate_pal_mode_50hz->setToolTip(
+      tr("Sets the Wii display mode to 50Hz (576i) instead of 60Hz "
```

Comment:
It is useful to repeat that this is for PAL games only here in the tooltip, in case someone glances over the PAL title. Also some variations can help with making it a bit more readable. Something like "Sets the Wii display mode to the standard 50Hz (576i) mode for PAL games."

Also I wasn't able to find a specific list, but the number of PAL games that do not support 50hz can be counted on one hand. If you'd like to mention it, please say something like "Almost all PAL titles use this mode."

All together, I recommend: 

>Sets the Wii display mode to the standard 50Hz (576i) mode for PAL games. Almost all PAL titles use this mode.

Reply:
I believe Wii autodetects EDTV/HDTV as I was playing SSBB on Wii just hours ago (after many years, it's still alive!) and when I tried running a demo classic inside SSBB it asked me it had to swith the display mode to 50hz, so I would replace the "use" with "support" as "use" might imply it's default irrespective of connection type or TV.

Reply:
Yes, it should say support rather than use. Though I can't recall it switching to 60 Hz automatically like you say.

Reply:
Allright, I assumed it's 60Hz because when I try to play trial versions of Masterpieces in SSBB it says it needs to switch to 50hz, , ofcourse that's not proof.

But I forgot to mention, I am connecting via COMPONENT cable (YPbPr) and I have checked right now to make sure and yes the HDTV (Samsung UE40D6750) is showing 720x480 60hz on the source label as the Wii warning screen is showing when it boots.

Reply:
You've probably changed the setting to 60 Hz at some point.

Reply:
Oops. Indeed. I forgot quite a bit I guess. Wii has 3 options and I have it set to EDTV/HDTV 480p which is 60hz. But there is also a 60hz 480i mode.

I think I heard dolphin doesn't emulate interlaced right. so those, but if the PAL60 (E60 in code) is the SYSCONF Wii native settings, which of the two is it?